.github/workflows/local.yml

```diff
@@ -55,8 +55,21 @@ jobs:
       uses: mxschmitt/action-tmate@v2
 
     - name: Build GraphScope
+      env:
+        OS : ${{ matrix.os }}
```

Comment:
Why do we still have the stage for building in this script?
The target of this renaming issue is to make the building process clear and easy:
A user uses this script to INSTALL dependencies ONLY.

Then, the one can only use a single command 
`make graphscope` to build the artifacts by himself.